README.md

```diff
@@ -0,0 +1,51 @@
+# SHiELD-wrapper
+
+This repository contains code for a Python-wrapped version of the public
+version of GFDL's SHiELD model.  It aims to be a drop-in replacement for
+[AI2's existing Python-wrapped version of
+FV3GFS](https://github.com/ai2cm/fv3gfs-fortran/tree/master/FV3/wrapper), which
```

Comment:
Might want to say "It is based on" instead of "It aims to be a drop-in replacement for" if we aren't using it as a drop-in replacement.

Reply:
Thanks, indeed this language was a bit too strong.  

Even when on par in terms of features, it will not be a drop-in replacement due to differences in configuration, i.e. configs for FV3GFS will not work as configs of SHiELD due to namelist parameter differences, differences in how the forcing files are treated, and names of diagnostics, among other things.